Traceback (most recent call last):
  File "./typthonCompiler.py", line 71, in <module>
    typechecker.typecheck(root)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 11, in typecheck
    return getattr(self, method)(node, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 55, in check_File
    child_st = self.typecheck(child, global_st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 11, in typecheck
    return getattr(self, method)(node, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 181, in check_StmtList
    stmtType = self.typecheck(stmt, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 11, in typecheck
    return getattr(self, method)(node, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 326, in check_FunctionDefn
    body_return_type = self.typecheck(node.body, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 11, in typecheck
    return getattr(self, method)(node, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 174, in check_StmtList
    retType = self.typecheck(stmt, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 11, in typecheck
    return getattr(self, method)(node, st)
  File "C:\Users\liams\project\sprint4\typthonTypeChecker.py", line 350, in check_RetStmt
    raise ParseError(
typthonSymbolTable.ParseError: ('Return type of str does not match with expected type int', 3)
